core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -2132,6 +2174,7 @@ public boolean rename(TachyonURI srcPath, TachyonURI dstPath) throws FileDoesNot
     synchronized (mRootLock) {
       Inode inode = getInode(srcPath);
       if (inode == null) {
+        mMasterSource.incRenameOps();
```

Comment:
sounds good to me. One alternative way might be calling mMasterSource.incRenameOps(); both in the beginning of rename(TachyonURI srcPath, TachyonURI dstPath), and rename(int fileId, TachyonURI dstPath), rather than here and renameInternal? It looks to me more consistent with other methods.